src/main/java/org/openmrs/addonindex/rest/AddOnController.java

```diff
@@ -40,6 +40,11 @@ public AddOnController(Index index) {
 	                                           @RequestParam(value = "q", required = false) String query) throws Exception {
 		return index.search(type, query);
 	}
+
+	@RequestMapping(method = RequestMethod.GET, value = "/api/v1/tags")
```

Comment:
I was initially planning to do that but then how do I call 2 functions under the same mapping or rather can we?